apps/api/src/app/topics/use-cases/filter-topics/filter-topics.use-case.ts

```diff
@@ -13,7 +13,7 @@ export class FilterTopicsUseCase {
   constructor(private topicRepository: TopicRepository) {}
 
   async execute(command: FilterTopicsCommand) {
-    const { pageSize = DEFAULT_TOPIC_LIMIT, page = 0 } = command;
+    const { pageSize = DEFAULT_TOPIC_LIMIT, page = 1 } = command;
```

Comment:
We do the move to natural count. Any user will think that first page is page 1, not page 0. Even if we are used to arrays.